pr_line_related_po_line/static/description/index.html

```diff
@@ -9,12 +9,9 @@ <h2 class="oe_slogan">
    PR Line related PO Line
   </h2>
   <p class="oe_mt32">
-   Add purchase_requisition_line_id field, it is id of purchase requisition line from where purchase
-order line is created, overwrite  the make_purchase_order method for add value of
-purchase_requisition_line_id to record purchase order line, it is help to make best inherit and
-modification of make_purchase_order method, as can be seen in
-purchase_requisition_line_description, purchase_requisition_line_analytic and
-purchase_requisition_requisitor modules.
+    This module adds a related purchase requisition line whenever a purchase order line is created. It also
+    overwrites the method to create a purchase order line to ad the id of a purchase requistition line to that
```

Comment:
```suggestion
    overwrites the method to create a purchase order line to ad the id of a purchase requisition line to that
```

But this part gets technical, someone without technical knowledge won't know what's a method.

I'd say something like "it also adds a new field to purchase order lines to link them to the created requisition lines.